imports/plugins/core/accounts/client/containers/accountsDashboardContainer.js

```diff
@@ -2,103 +2,104 @@ import { compose, withProps } from "recompose";
 import Alert from "sweetalert2";
 import { registerComponent, composeWithTracker, withIsAdmin } from "@reactioncommerce/reaction-components";
 import { Meteor } from "meteor/meteor";
+import getOpaqueIds from "/imports/plugins/core/core/client/util/getOpaqueIds";
+import simpleGraphQLClient from "/imports/plugins/core/graphql/lib/helpers/simpleClient";
 import { Accounts, Groups } from "/lib/collections";
 import { Reaction, i18next } from "/client/api";
 import AccountsDashboard from "../components/accountsDashboard";
+import addAccountToGroupMutation from "./addAccountToGroup.graphql";
+import removeAccountFromGroupMutation from "./removeAccountFromGroup.graphql";
+
+const addAccountToGroupMutate = simpleGraphQLClient.createMutationFunction(addAccountToGroupMutation);
```

Comment:
Just making a note to refactor this without `simpleGraphQLClient` in the future.

Reply:
Yeah I didn't want to go too far down the rabbit hole.